pkg/kvstore/etcd.go

```diff
@@ -406,6 +426,7 @@ func getEPVersion(c client.Maintenance, etcdEP string, timeout time.Duration) (*
 // function should be run whenever the etcd client is connected for the first
 // time and whenever the session is renewed.
 func (e *etcdClient) checkMinVersion() error {
+	e.limiter.Wait(ctx.Background())
```

Comment:
`Endpoints()` is not a kvstore operation I assume?

Reply:
You're right, good catch.